pkg/lib/listing.less

```diff
@@ -98,6 +98,30 @@ tbody.open > .listing-ct-panel {
 tbody.open > .listing-ct-panel > td > .listing-ct-body {
     border: none;
     overflow: auto;
+    max-height: calc(80vh - 5rem);
+    position: relative;
+
+    /* FIXME: these two lines are for debugging only. Remove! */
+    .machines-disks { display: inline; }
+    .machines-disks-source-value { height: 900vh; }
```

Comment:
This messes up the height of all the rows except the first in the VM Disks subtab.
The comment on line 104 says it's only for debugging, so these 3 rows needs to go before we can land this.